src/components/tables/extras/TableHeader.vue

```diff
@@ -12,7 +12,7 @@
           variant="outlined"
           dark
           class="mb-2 text--primary"
-          :to="route"
+          :to="`/${route}`"
```

Comment:
cambiar path por name para acceder a la ruta

Reply:
No entendí

Reply:
:to="{ name: 'nombre-de-la-ruta' }"